### DISCUSSION TITLE
Add findOrFail

### DISCUSSION THREAD

**Discussion by francoism90** [spatie/laravel-prefixed-ids#10] [Ideas]
Laravel offers `Model::findOrFail(..)`, this is really useful as you don't need to write logic for it when the value is null.

Would you consider adding it? I'll try to create a PR, but would like to know if this feature is wanted first. :)

**Comment by riasvdv** [comment]
Sure! You can call it `findByPrefixedIdOrFail` to keep it in line with the naming

**Comment by francoism90** [comment]
@riasvdv See #11 , hopefully it's Ok, please let me know if any changes are needed. :)
